starwars/src/App.css

```diff
@@ -1,7 +1,9 @@
 .App {
   text-align: center;
+  margin: 0 auto;
```

Comment:
This is good! I usually use the App as my "container" and give it the same styling I would the container divs we've used in the past.